sdk/botservice/Microsoft.Azure.Management.BotService/src/Generated/Models/MsTeamsChannelProperties.cs

```diff
@@ -31,23 +31,14 @@ public MsTeamsChannelProperties()
         /// </summary>
         /// <param name="isEnabled">Whether this channel is enabled for the
         /// bot</param>
-        /// <param name="enableMessaging">Enable messaging for Microsoft Teams
-        /// channel</param>
-        /// <param name="enableMediaCards">Enable media cards for Microsoft
-        /// Teams channel</param>
-        /// <param name="enableVideo">Enable video for Microsoft Teams
-        /// channel</param>
         /// <param name="enableCalling">Enable calling for Microsoft Teams
         /// channel</param>
-        /// <param name="callMode">Enable messaging for Microsoft Teams
-        /// channel</param>
-        public MsTeamsChannelProperties(bool isEnabled, bool? enableMessaging = default(bool?), bool? enableMediaCards = default(bool?), bool? enableVideo = default(bool?), bool? enableCalling = default(bool?), string callMode = default(string))
+        /// <param name="callingWebHook">Webhook for Microsoft Teams channel
+        /// calls</param>
+        public MsTeamsChannelProperties(bool isEnabled, bool? enableCalling = default(bool?), string callingWebHook = default(string))
```

Comment:
This will result in a breaking change do you have the link to the spec change associated with this?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/3478 this has been changed for over 2 years